bridges/scalajs/src/main/scala/bloop/scalajs/JsBridge.scala

```diff
@@ -38,40 +40,31 @@ object JsBridge {
   private def findIrFiles(path: Path): List[Path] =
     Files.walk(path).iterator().asScala.filter(isIrFile).toList
 
-  private def toIRJar(jar: Path) =
+  private def toIrJar(jar: Path) =
     IRContainer.Jar(new FileVirtualBinaryFile(jar.toFile) with VirtualJarFile)
 
   def link(
       config: JsConfig,
       project: Project,
       mainClass: String,
+      target: File,
       logger: BloopLogger
-  ): Path = {
+  ): Unit = {
```

Comment:
Why? I think it's better to return the `target`, even if it's already one of the inputs.

Reply:
Because we are passing in the path to the function now. Otherwise, we would make the assumption that the linker may save the file to a different path, which goes counter to the idea of having a parameter like `--output`.